scapy/layers/radius.py

```diff
@@ -1154,3 +1154,5 @@ def post_build(self, p, pay):
 bind_layers(UDP, Radius, dport=1812)
 bind_layers(UDP, Radius, sport=1813)
 bind_layers(UDP, Radius, dport=1813)
+bind_layers(UDP, Radius, sport=3799)
+bind_layers(UDP, Radius, dport=3799)
```

Comment:
Could you rather use
```
bind_bottom_up(UDP, Radius, sport=3799)
bind_bottom_up(UDP, Radius, dport=3799)
bind_layers(UDP, Radius, sport=3799, dport=3799)
```
And update the ones above ?

You only need a single `bind_layers` in total, which will be the default when building the packets. All others are only used for dissection

Thanks

Reply:
Ah! Got it, thank you @gpotter2 . I updated the pr.

Reply:
@gpotter2 so, I did some tests as you suggested but didn't work.

e.g:

1. Not work.

```python
bind_bottom_up(UDP, Radius, sport=1812)
bind_bottom_up(UDP, Radius, dport=1812)
bind_bottom_up(UDP, Radius, sport=1813)
bind_bottom_up(UDP, Radius, dport=1813)
bind_bottom_up(UDP, Radius, sport=3799)
bind_bottom_up(UDP, Radius, dport=3799)
bind_layers(UDP, Radius, sport=3799, dport=3799)
```

But, that one works pretty well.

```python
bind_layers(UDP, Radius, sport=1812)
bind_layers(UDP, Radius, dport=1812)
bind_layers(UDP, Radius, sport=1813)
bind_layers(UDP, Radius, dport=1813)
bind_layers(UDP, Radius, sport=3799)
bind_layers(UDP, Radius, dport=3799)
```

btw, I noticed other layers using multiple `bind_layers()` for different ports as the `dns.py`. therefore, what do you suggest?

Reply:
Ah! Got it, thank you @gpotter2 . I updated the pr.